src/App.jsx

```diff
@@ -1,32 +1,17 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import { Row, Col } from 'reactstrap';
 import Login from './components/login';
-import ReportPage from './components/reportPage';
-import Navbar from './components/layout/navbar';
-import Sidebar from './components/layout/sidebar';
-import Footer from './components/layout/footer';
+import ReportPage from './components/ReportPage';
 
 class App extends React.PureComponent {
   render() {
     return (
       <div>
         <BrowserRouter>
-          <div>
-            <Navbar />
-            <Switch>
-              <Route exact path="/login" component={Login} />
-              <Row>
-                <Col sm={2}>
-                  <Sidebar />
-                </Col>
-                <Col sm={10}>
-                  <Route exact path="/" component={ReportPage} />
-                </Col>
-              </Row>
-            </Switch>
-            <Footer />
-          </div>
+          <Switch>
+            <Route exact path="/login" component={Login} />
+            <Route exact path="/report" component={ReportPage} />
```

Comment:
Is report page the main page in this application? If so, I would suggest the path to be `'/'`

Reply:
Yp, the report page is the main page of the application and I also agree that `/` would be a better path name. I would update the code.

Reply:
Done!